web3/utils/__init__.py

```diff
@@ -25,3 +24,6 @@
     transactions,
     validation,
 )
+from web3.tools import (  # noqa: F401
```

Comment:
We've been establishing the `library.tools` module as a place that non-essential library functionality goes, and the general rule is that we should never import from `library.tools` anywhere outside of that module.

Instead of adding this import, can you update all of the places that use this in the tests to import from the new location.

Reply:
I think I get you, so I see ~7 places this is imported, ill be updating those to `web3.tool.hypothesis` rather than just `hypothesis` so its no longer needed here, right? 
I can do that 👍